examples/smart-escrows/ledger_sqn/README.md

```diff
@@ -0,0 +1,62 @@
+# Ledger Sequence Escrow
+
+This WebAssembly module implements a simple ledger sequence-based escrow finish condition. It checks whether the current ledger sequence number is greater than 5 before allowing the escrow to be finished.
```

Comment:
```suggestion
This WebAssembly module implements a simple ledger sequence-based escrow finish condition.
```

Consider leaving the summary short, and deferring the mechanics to "How it works", which says this part again (alternatively, ignore this suggestion if you like things the way they are)